tests/fixtures/gandi-no-changes.json

```diff
@@ -0,0 +1,127 @@
+[
+      {
+        "rrset_type": "A",
+        "rrset_ttl": 300,
+        "rrset_name": "",
+        "rrset_href": "https://api.gandi.net/v5/livedns/domains/reductioncarbone.fr/records/%40/A",
```

Comment:
You'll want to swap this to unit.tests instead of reductioncarbone.fr

Reply:
Fixed.